Yoast/javascript
# Add support for irregular Italian participles

## Summary


This PR can be summarized in the following changelog entry:

* [yoastseo] Adds support for irregular Italian participles by canonicalizing verb stems.

## Relevant technical choices:

* In some cases, forms that aren't part of the verb paradigm get recognized as such. These concern the following words (judged by a quick glance, perhaps not an exhaustivel ist): 
avvoltati
corsia
corsivo
coltivate
lettere
puntata
trattare/trattato
However, since these still get stemmed to the same form, the actual consequence of this is small because it just means that it's theoretically possible that they get merged with the forms of a verb paradigm for the purposes of keyphrase recognition (e.g. _lettere_, "letter", gets merged with the paradigm _leggere_, "to read", because that verb has the participle _letto_. So a text about reading letters is one of the few real-life examples where you will notice it.).


## Test instructions

This PR can be tested by following these steps:

* Add forms of the verb paradigms for which canonical stems have added in `stemSpec` and make sure they get the same stem. 
* Add different forms of the verb paradigms for which canonical stems have added in the content analysis app and confirm that they get recognized if you add a form of that verb in the keyphrase.

## Impact check

* This PR affects the following parts of the plugin, which may require extra testing:
  *

## UI changes

## Quality assurance


Fixes https://yoast.atlassian.net/browse/LIN-423
